airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/ConnectorBreakingChanges.yaml

```diff
@@ -25,6 +25,13 @@ definitions:
       message:
         description: Descriptive message detailing the breaking change.
         type: string
+      deadlineAction:
```

Comment:
This having a default value means that we don't _have_ to backfill the existing connectors, right?

Reply:
@alafanechere do we want to backfill, or nah? 

If we don't backfill, do we want a followup PR with metadata schema docs update? :D

Reply:
Good question.
As registry entries are generated on publish, we would have to backfill already published metadata / registry entries for this kind of change if we want to expose the default value in the registry.

IMO the default value should be handled platform side as it's a global behavior that could change between OSS and Cloud.
When we'll work on consuming this `deadlineAction` in the platform we can make it set `auto_upgrade` when `deadlineAction` is null.

**So my recommendation is to  set a default value and keep this field optional.**
* No backfill will be needed
* Platform will use the metadata defined `deadlineAction` if it's declared or fallback to its default deadlineAction (like `auto_upgrade`).

Wdyt?

Reply:
I'm okay with removing the default value if it's going to be set on the platform.